values.yaml

```diff
@@ -43,6 +43,8 @@ global:
     # install: install kafka streaming platform
     install: false
 
+  # persistent-volume and associated claims configurations
+
   # guarantee the data we are expecting is mounted appropriately
```

Comment:
should there be a space between these comments?

Reply:
Space removed between comments